thrust/system/cuda/detail/core/triple_chevron_launch.h

```diff
@@ -29,8 +29,10 @@
 #include <thrust/detail/config.h>
 #include <thrust/system/cuda/detail/core/alignment.h>
 #include <thrust/system/cuda/detail/guarded_cuda_runtime_api.h>
-#include <cassert>
 
+#include <nv/target>
+
+#include <cassert>
```

Comment:
Technically you need 
```suggestion
#include <cassert>
#include <cstddef>
```

because of `std::size_t`